tests/unit/tck/handlers_test.py

```
@@ -0,0 +1,313 @@
+            raise ValueError("Something went wrong")
+
+        with pytest.raises(JsonRpcError) as excinfo:
+            dispatch("crash_method", {}, None)
+        
+        assert excinfo.value.code == INTERNAL_ERROR
+        assert "Something went wrong" in excinfo.value.data
+
+    def test_dispatch_converts_precheck_error_to_hiero_error(self):
+        """Test that dispatch converts PrecheckError to HIERO_ERROR."""
+        @register_handler("precheck_method")
+        def precheck_handler(_params):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `precheck_handler`

Add return type annotation: `NoReturn`

(ANN202)